### PR TITLE
f-vue-icons@v2.4.0 – Add MoreVertical Icon

### PR DESCRIPTION
**Pull request by nehadhanuka-justeat**
- Updated the f-icon package
- Added the MoreVerticalIcon type

### CODE REVIEW

File: packages/tools/f-vue-icons/CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,13 @@
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html)
 
+v2.3.2
```
Comment (ashleynolan) [inline packages/tools/f-vue-icons/CHANGELOG.md]: Should bump this to `v2.4.0`

**Review by ashleynolan — CHANGES_REQUESTED** [review]
You'll need to update the `package.json` version of the `f-vue-icons` to `2.4.0` to publish the package.

Also, if you pull in latest master, the bot error you're seeing should disappear when you push that commit (that was fixed yesterday)

**Review by ashleynolan — DISMISSED** [review]

**Review by FayeCarter — DISMISSED** [review]

**Comment by kevinrodrigues** [comment]
Missing the yarn.lock file

**Review by kevinrodrigues — DISMISSED** [review]

**Review by DamianMullins — DISMISSED** [review]

**Comment by nehadhanuka-justeat** [comment]
Can this be merged pls?

File: yarn.lock [hunk 2]
```diff
@@ -1939,10 +1939,10 @@
   dependencies:
     vue-i18n "8.0.0"
 
-"@justeat/f-icons@3.3.1":
-  version "3.3.1"
-  resolved "https://registry.yarnpkg.com/@justeat/f-icons/-/f-icons-3.3.1.tgz#28b7fb405ffc0d5285391449faa6806668b3f6ed"
-  integrity sha512-dj4jwA5vV1lJJBz7vfwPGobdY0UNP27oMotxhNPGQMbcDHDlOMIIKs+tKnAEdOQS7+wz2WSnk+CCcLNP2nzcwA==
+"@justeat/f-icons@3.4.0":
+  version "3.4.0"
+  resolved "http://packages.je-labs.com/npm/private-npm/@justeat/f-icons/-/f-icons-3.4.0.tgz#dbdeb97c562926dc1c4b98685b25203a97dbd5ff"
```
Comment (kevinrodrigues) [inline yarn.lock]: Looks like the build is failing because of the private npm reference.

**Comment by nehadhanuka-justeat** [comment]
The build error is UnAuthorized - any idea why so?

**Review by ashleynolan — APPROVED** [review]

**Review by simonsandersje — APPROVED** [review]

**Pull request merged**